jsx-runtime/src/index.js

```diff
@@ -1,6 +1,5 @@
 import { options, Fragment } from 'preact';
-import { encodeEntities } from './utils';
-import { IS_NON_DIMENSIONAL } from '../../src/constants';
+import { encodeEntities, IS_NON_DIMENSIONAL } from './utils';
```

Comment:
Would it be too problematic to make this compat-only? Moving it out of core is good for marketing and whatnot but I'd prefer to go a step further if at all possible

Reply:
I will defer this to @marvinhagemeister as Deno uses the JSX template transform exclusively atm